src/main/scala/codacy/plugins/DockerTest.scala

```diff
@@ -50,18 +56,19 @@ object DockerTest {
   }
 
   private def run(plugin: DockerPlugin, testSources: Seq[Path], test: ITest, testRequest: String, dockerImageName: String, optArgs: Seq[String]): Boolean = {
-    if ((allTestKey :+ test.opt).contains(testRequest)) {
-      test.run(plugin, testSources, dockerImageName, optArgs) match {
-        case true =>
-          Printer.green(s"[Success] ${test.getClass.getSimpleName}")
-          true
-        case _ =>
-          Printer.red(s"[Failure] ${test.getClass.getSimpleName}")
-          false
-      }
-    } else {
-      // this test was not selected
-      true
+    config.get(testRequest) match{
```

Comment:
i had pushed rearrange code, i thought this would do everything (format, optimize imports etc) but i was wrong... 
do you always press all of them when you format?
anyway, it's done now i hope